src/forms/CalendarComponent.jsx

```diff
@@ -86,7 +87,7 @@ export class CalendarComponent extends React.Component {
 		return (
 			<span>
 				{label && (
-					<label htmlFor={id || name} className={classNames.label}>
+					<label htmlFor={id || name} className={classNames.label} data-requiredtext={requiredText && `(${requiredText})`}>
```

Comment:
I'd remove the parens here; requiring the parens in the translation string will provide context to the translators and avoid any weird situations where parens may not make sense in a certain language.